content/en/_index.html

```diff
@@ -124,19 +124,19 @@ <h6>Our lives are colorful, why not our coding editor</h6>
 						</div>
 					</div>
 				</div>
-				<div class="tab-pane fade" id="customizable" role="tabpanel" aria-labelledby="contact-tab">
+				<div class="tab-pane fade" id="linting" role="tabpanel" aria-labelledby="contact-tab">
 					<div class="row made_life_text">
 						<div class="col-lg-6">
 							<div class="left_side_text">
-								<h3>Why change your habits?<br />We can change the shortcuts</h3>
-								<h6>You can use the shortcuts you have already got used to</h6>
-								<p>We understand that getting used to new keybindings is more difficult than getting used to a new language, so we have added options for you to choose your own keybindings for actions in the editor.</p>
+								<h3>Errors and Warnings?<br />We lint them as you code</h3>
+								<h6>Saving your precious contest time by linting code as you type</h6>
+								<p>Using Language Server program we provide realtime errors, warnings, informations and hints about your code on the go in the editor. It is same like VSCode's and Atom's Intellisense except it is fast.</p>
```

Comment:
```suggestion
								<p>Using Language Server program, we provide real-time linting for errors and warnings in your code, just like Intellisense in VSCode and Atom.</p>
```

"errors and warnings" is enough and clear.

We shouldn't guarantee it's fast, because that depends on the language server program.

Reply:
Okay!

Reply:
I merged it hastening because @nitishk72 will be working on making the site more responsive and some other improvements.